### PR TITLE
Update Model Testing

### PR DESCRIPTION
**Pull request by CheezItMan**
## Description
While I was looking at controller testing I figured I'd review the model testing section too.  I just updated it for Rails 5.
- replaced `rake test` with the `rails test` equivalent
- took out the table with rake testing commands and left in the Rails 5 table. 
- simplified the setup instructions and marked minitest-reporters as optional.

## Todos
None on this document

## Feedback Requested
Feedback is welcome

### CODE REVIEW

File: 08-rails/00-unit-testing-models-specs.md [hunk 1]
```diff
@@ -206,10 +172,10 @@ Also check our _expectations_. They look like Minitest _expectations_ because th
 __Question: Is the test above a _positive_ or _negative_ test? What would the other test look like?__
 
 ## Running tests
-[The Rails Guide on testing](http://guides.rubyonrails.org/testing.html#rake-tasks-for-running-your-tests) has a specific section for how to run tests that's definitely worth reading. The short version is that from the project root, we can run all of our tests with `rake test`. If you want to run just the model tests, run `rake test:models`. The output will probably look pretty familiar by now:
+[The Rails Guide on testing](http://guides.rubyonrails.org/testing.html#rake-tasks-for-running-your-tests) has a specific section for how to run tests that's definitely worth reading. The short version is that from the project root, we can run all of our tests with `rails test`. If you want to run just the model tests, run `rails test/models`. The output will probably look pretty familiar by now:
```
Comment (Hamled) [inline 08-rails/00-unit-testing-models-specs.md]: We should update the URL for the Rails Guide link to use http://guides.rubyonrails.org/testing.html#the-rails-test-runner .

Comment (Hamled) [inline 08-rails/00-unit-testing-models-specs.md]: Also `rails test/models` I think needs to be `rails test test/models`.

File: 08-rails/00-unit-testing-models-specs.md [hunk 2]
```diff
@@ -206,10 +172,10 @@ Also check our _expectations_. They look like Minitest _expectations_ because th
...
 
 ```bash
-$ rake test:models
+$ rails test/models
```
Comment (Hamled) [inline 08-rails/00-unit-testing-models-specs.md]: Same here, I think this needs to be `$ rails test test/models`

File: 08-rails/00-unit-testing-models-specs.md [hunk 3]
```diff
@@ -41,63 +41,28 @@ This means when Rake or Rails runs the test task, it will use the minitest-rails
 
 Then run `bundle install`
 
-#### Installing Minitest Spec
+#### Step 2:  Installing Minitest Spec
 
-Then we will ask Rails to modify test_helper.rb to use spec style describe and it blocks with `rails generate minitest:install`  It will ask you if you want to overwrite the test_\_helper.rb file.  Answer yes (Y).
+Then we will ask Rails to modify test_helper.rb to use spec style describe and it blocks with `rails generate minitest:install`  It will ask you if you want to overwrite the test_\_helper.rb file.  Answer yes **(Y)**.
```
Comment (Hamled) [inline 08-rails/00-unit-testing-models-specs.md]: The "test_\\_helper.rb" bit at the end of the second sentence is giving us two underscores. I think we only need "test_helper.rb" there.

**Comment by CheezItMan** [comment]
Changes made

File: 08-rails/00-unit-testing-models-specs.md [hunk 4]
```diff
@@ -206,10 +172,10 @@ Also check our _expectations_. They look like Minitest _expectations_ because th
 __Question: Is the test above a _positive_ or _negative_ test? What would the other test look like?__
 
 ## Running tests
-[The Rails Guide on testing](http://guides.rubyonrails.org/testing.html#rake-tasks-for-running-your-tests) has a specific section for how to run tests that's definitely worth reading. The short version is that from the project root, we can run all of our tests with `rake test`. If you want to run just the model tests, run `rake test:models`. The output will probably look pretty familiar by now:
+[The Rails Guide on testing](http://guides.rubyonrails.org/testing.html#the-rails-test-runner) has a specific section for how to run tests that's definitely worth reading. The short version is that from the project root, we can run all of our tests with `rails test`. If you want to run just the model tests, run `rails test/models`. The output will probably look pretty familiar by now:
```
Comment (Hamled) [inline 08-rails/00-unit-testing-models-specs.md]: `rails test/models` should be `rails test test/models`.

File: 08-rails/00-unit-testing-models-specs.md [hunk 5]
```diff
@@ -206,10 +172,10 @@ Also check our _expectations_. They look like Minitest _expectations_ because th
...
 
 ```bash
-$ rake test:models
+$ rails test/models
```
Comment (Hamled) [inline 08-rails/00-unit-testing-models-specs.md]: `rails test/models` should be `rails test test/models`.

**Comment by Hamled** [comment]
Okay, it looks like we need to add back in the section about updating `config/application.rb` with the following lines:
```ruby
config.generators do |g|
  g.test_framework :minitest, spec: true
end
```

unless there's another way to force spec-style in generated tests.

**Comment by Hamled** [comment]
@CheezItMan what's the other setting you were using? Would it make more sense to have them use that setting?

**Comment by CheezItMan** [comment]
Unfortunately I'd setup in December a bash script which ran in place of rails new, which created a new rails project with a template.rb and promptly forgotten about it. 

More involved than I'd like and I'd rather have students working with rails, and minitest-rails basic instead.

File: 08-rails/00-unit-testing-models-specs.md [hunk 6]
```diff
@@ -182,7 +182,7 @@ Also check our _expectations_. They look like Minitest _expectations_ because th
 __Question: Is the test above a _positive_ or _negative_ test? What would the other test look like?__
 
 ## Running tests
-[The Rails Guide on testing](http://guides.rubyonrails.org/testing.html#the-rails-test-runner) has a specific section for how to run tests that's definitely worth reading. The short version is that from the project root, we can run all of our tests with `rails test`. If you want to run just the model tests, run `rails test/models`. The output will probably look pretty familiar by now:
+[The Rails Guide on testing](http://guides.rubyonrails.org/testing.html#the-rails-test-runner) has a specific section for how to run tests that's definitely worth reading. The short version is that from the project root, we can run all of our tests with `rails test`. If you want to run just the model tests, run `rails test test/models`. The output will probably look pretty familiar by now:
 
 ```bash
 $ rails test/models
```
Comment (Hamled) [inline 08-rails/00-unit-testing-models-specs.md]: This should also be `$ rails test test/models`.

**Pull request merged**